### PR TITLE
Added the function assert_torch_installed() for message consistency.

### PR DESCRIPTION
**Pull request by ghost**
Repeated messages is not a good practice as it can cause difficulty and inconsistency when somebody wants to change the message line. Consider adding another function that does the assertion job.

### CODE REVIEW

File: hummingbird/ml/convert.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 from ._parse import parse_sklearn_api_model, parse_onnx_api_model, parse_sparkml_api_model
 from ._topology import convert as topology_converter
 from ._utils import (
-    torch_installed,
+    assert_torch_installed,
```
Comment (ksaur) [inline hummingbird/ml/convert.py:18]: Can you please also change the other `_installed` (Ex lightgbm/xgeboot/pandas) to be prefixed with `assert` for consistency?

**Review by ksaur — CHANGES_REQUESTED** [review]
Thanks, you are right, this is a better practice! :)

For consistency, can you please also change the others to match (see my comment below) and also throughout `convert.py` (ex `lightgbm_installed` on line 117)

File: hummingbird/ml/convert.py (hunk 1 above)
Comment (ghost) [reply hummingbird/ml/convert.py:18]: @ksaur Done!

Comment (ghost) [reply hummingbird/ml/convert.py:18]: I've also fixed some spelling errors along the way. In this case, the word `XGBoost` and `install` was incorrectly spelled and capitalized!
image

**Review by ksaur — APPROVED** [review]
Thanks for the additional fixes! :)

I think there's just a whitespace lint thing, then we can merge it.

**Comment by ghost** [comment]
@ksaur Oh, really sorry for that! It looks ok now. 🙂

**Pull request merged**